src/Patch/HTMLElementSubclasses.js

```diff
@@ -0,0 +1,69 @@
+import Native from './Native.js';
+import CustomElementInternals from '../CustomElementInternals.js';
+import CEState from '../CustomElementState.js';
+import AlreadyConstructedMarker from '../AlreadyConstructedMarker.js';
+
+/**
+ * @param {!CustomElementInternals} internals
+ */
+export default function(internals) {
```

Comment:
Browsers currently throw errors if you try to subclass native DOM elements. So we patch each of the DOM element constructors so that they don't throw errors.